src/language/functions.md

```diff
@@ -500,6 +500,26 @@ Iterable<int> naturalsDownFrom(int n) sync* {
 }
 ```
 
+## External functions {#external}
+
+Declare a function `external` if its body is defined separately from its
+declaration in your Dart program. 
+
+```dart
+external void someFunc(int i);
+```
+
+External functions can be top-level functions, [instance methods][],
+[getters or setters][], or [non-redirecting constructors][].
+[Instance variables][] can be `external` too,
+and essentially equate to a getter or setter on the external value. 
+
+The `external` keyword usually appears in interop contexts,
+to introduce type information for foreign functions or values,
+and to make them usable in Dart. Implementation and usage is
+heavily platform specific, so check out the interop docs on, for example,
+[C][] or [JavaScript][] to learn more.
```

Comment:
This "C" in the raw markdown looks funny for some reason 😆. It's so short. Good thing Dart has 4 letters 👀

Reply:
> Good thing Dart has 4 letters 👀

Clearly the superior language 😆